grouper/fe/handlers.py

```diff
@@ -821,6 +821,12 @@ def post(self, audit_id):
             return self.forbidden()
 
         audit = self.session.query(Audit).filter(Audit.id == audit_id).one()
+
+        # only owners can complete
+        owner_ids = {member.id for n,member in audit.group.my_owners().items()}
```

Comment:
i was intending to help document my_owners() since it doesn't do what i expect but i guess this was the totally wrong approach to that. :)

this is also a set vs a dict.

i updated to use `values()` instead so there's no confusion about `n`.